utility/src/main/java/org/oppia/util/parser/ImageParsingAnnotations.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.util.parser
+
+import javax.inject.Qualifier
+
+/** Corresponds to the default GCS Resource Bucket Name. */
+@Qualifier annotation class DefaultGCSResource
+
+/** Corresponds to the default Gcs Prefix. */
+@Qualifier annotation class DefaultGcsPrefix
+
+/** Corresponds to the default Image Download Url Template. */
+@Qualifier annotation class ImageDownloadUrlTemplate
```

Comment:
I think it's definitely worth documenting that strings bound to this template will have 3 format variables in them, and what those variables should correspond to.

Reply:
@veena14cs This conversation ha not been addressed.

Reply:
added doc comment.